demos/fab.scss

```diff
@@ -26,3 +26,32 @@
 .demo-fab-icon-size {
   @include mdc-fab-icon-size(36px);
 }
+
+.demo-figure-fab-extended {
+  width: 100%;
+  padding: 0;
+  margin: 1em 40px;
+  box-sizing: border-box;
+}
+
+.demo-fab-extended-fluid {
+  @include mdc-fab-extended-fluid();
```

Comment:
`@include  mdc-fab-extended-fluid` - don't need the parens if there are no args

Reply:
Done.